OPHD/States/MapViewStateUi.cpp

```diff
@@ -565,13 +565,13 @@ void MapViewState::fileIoAction(const std::string& filePath, FileIo::FileOperati
 			doNonFatalErrorMessage("Load Failed", e.what());
 			return;
 		}
+		mFileIoDialog.hide();
 	}
-	else
+	else if (fileOp == FileIo::FileOperation::FILE_SAVE)
 	{
 		save(constants::SAVE_GAME_PATH + filePath + ".xml");
+		mFileIoDialog.hide();
 	}
-
-	mFileIoDialog.hide();
 }
```

Comment:
Looks like the changes here are not strictly needed after backing out of the new return type.